### PR TITLE
Add option to the mod menu for compatibillity

### PR DESCRIPTION
**Pull request by XenoTheStrange**
Users might want to render maps which aren't technically player homes. I know I do!

I've been using DeepRim for awhile and couldn't get it to work and render images for underground maps, so I looked in the source to find out why. Turns out to be a single if-statement which returns -_-.
Now it'll check for a mod config option which allows it to capture images of other maps.

### CODE REVIEW

**Comment by XenoTheStrange** [comment]
I'm thinking of adding an option that will also sort screenshots based on the name (ToString) of the map they're on so things are easier to navigate. Rn it mixes up different maps, giving them the same name but adding "alt-1" as a suffix.

**Comment by XenoTheStrange** [comment]
Hmmmmmm I might just exclude the .csproj file for sake of conflicts. This is the versoin of progress renderer that's on steam rn, right?

**Comment by XenoTheStrange** [comment]
Removed .csproj files from the commit

**Comment by XenoTheStrange** [comment]
I smushed the edits into one little non-conflicting commit.
I am not very good at the github.

**Comment by XenoTheStrange** [comment]
After rendering the mod yanks you to another map sometimes which is super annoying. Might look into that.

**Comment by XenoTheStrange** [comment]
image

**Comment by XenoTheStrange** [comment]
Working on fixing race condition that causes the map to switch to a random rendered one after rendering....

**Comment by XenoTheStrange** [comment]
image

After the first couple of renders it just _doesn't_ change the current map back to the remembered map, despite running the code which would do so: `Current.Game.CurrentMap = rememberedMap;`

**Comment by XenoTheStrange** [comment]
The problem of it switching to a random map is due to several instances of the DoRender function running at the same time, so while one changes the map to render it the other is saving _that_ map as the one to return to instead of the proper one:
image

**Comment by XenoTheStrange** [comment]
This error actually causes problems: It will make duplicate captures of some maps while ignoring others.
Pretty sure this problem applies whenever there are more than 2 maps being captured.

**Comment by XenoTheStrange** [comment]
I submitted an issue with the error I encountered, but beyond that this pull is ready.

**Comment by XenoTheStrange** [comment]
.... Merged your bump to 1.5 that happened to occur recently. All done.

File: Common/Languages/English/Keyed/All.xml [hunk 1]
```diff
@@ -36,6 +36,8 @@
 	<LPR_RenderFeedback_Window>Popup</LPR_RenderFeedback_Window>
 	<LPR_SettingsRenderSettingsLabel>Render settings</LPR_SettingsRenderSettingsLabel>
 	<LPR_SettingsRenderSettingsDescription>Configure some optional graphical elements that should be included in the rendered images. The map will always be rendered while some temporary parts like your current selection, targeters or ui text and the mouse cursor are never included.</LPR_SettingsRenderSettingsDescription>
+	<LPR_SettingsRenderNonPlayerHomes>Non-PlayerHome Maps</LPR_SettingsRenderNonPlayerHomes>
```
Comment (JonathanTroyer) [inline Common/Languages/English/Keyed/All.xml:39]: Change to "Non-colony maps"

**Review by JonathanTroyer — CHANGES_REQUESTED** [review]
Looks great besides that one label. Very good first PR, thank you!

File: Common/Languages/English/Keyed/All.xml (hunk 1 above)
Comment (XenoTheStrange) [reply Common/Languages/English/Keyed/All.xml]: Done

**Review by JonathanTroyer — APPROVED** [review]

**Pull request merged**